### PR TITLE
Consistent urls.py naming #199

### PR DESCRIPTION
**Pull request by theSage21**
Consistent urls.py naming. A new pull request as I re-forked.
All tests pass and flake8 gives no errors

### CODE REVIEW

**Comment by theSage21** [comment]
@kracekumar Please merge this. It is merge compatible I think

File: junction/urls.py [hunk 1]
```diff
@@ -40,7 +39,7 @@
     url(r'^faq/$', TemplateView.as_view(template_name='static-content/faq.html',), name='faq-static'),
 
     # Proposals as conference home page. TODO: Needs to be enhanced
```
Comment (kracekumar) [inline junction/urls.py]: Any reason to have `--` ?

Comment (theSage21) [inline junction/urls.py]: Not my work.  Feel free to remove.

Comment (theskumar) [inline junction/urls.py]: @kracekumar `--` was added to identify the conference view, which doesn't exist at the moment. The conference view, right now just redirect to it's proposal view.

More explaination:

Routes of the form `/<conference>/proposals` works fine, but when you just do `/<conference>/` where `<conference>` is a slug, the urls of the form `/about/` or `/privacy/` can be easily confused as a conference name. `/<conference>--` pattern was added just for the sake of recognising a conference homepage.

File: junction/urls.py [hunk 2]
```diff
@@ -26,9 +27,7 @@
     url('^markdown/', include('django_markdown.urls')),
 
     # Proposals related
-    url(r'^(?P<conference_slug>[\w-]+)/proposals/', include('junction.proposals.proposal_urls')),
-    url(r'^(?P<conference_slug>[\w-]+)/proposal-comments/', include('junction.proposals.comment_urls')),
-    url(r'^(?P<conference_slug>[\w-]+)/dashboard/', 'junction.proposals.views.dashboard', name='proposal-dashboard'),
+    url(r'^(?P<conference_slug>[\w-]+)/proposal/', include('junction.proposals.urls')),
```
Comment (kracekumar) [inline junction/urls.py]: This was renamed from `proposals` to `proposal`, which people noticed immediately https://twitter.com/indradhanush92/status/599982902558883842

**Comment by theSage21** [comment]
Sorry, did not predict that people had already saved urls.
Made sense to me "proposal/comments/" and "proposal/" instead of proposals/comments.
Do you want me to change it? I hope no one saved the one without the 's'

**Comment by kracekumar** [comment]
The issue is people tweet urls and memorize :-) I fixed it in latest master and deployed. Just check the comments urls.

**Comment by theSage21** [comment]
i only altered the proposals to proposal. also the proposal-comments/ became proposal/comments
Everything else was unaltered

**Comment by kracekumar** [comment]
Cool.

**Comment by theskumar** [comment]
@kracekumar it make sense for us to priorities on the integration/functional tests from now on. As the time will pass, we are going to loose control of the things very soon.

**Comment by kracekumar** [comment]
Agreed. Let's start that in parallel.

**Pull request merged**